corgi/collectors/errata_tool.py

```
@@ -261,3 +262,15 @@ def _parse_module(
                 # map modular_rpms to build_ids. In slow_save_errata these are only saved as
                 # meta_attr so, let's just save the entire list for each build_id
                 variant_to_component_map[variant].append({build_id: modular_rpms})
+
+    def get_erratum_notes(self, erratum_id: int):
+        # Get the contents of the "Notes" (formerly "How to test") field from an erratum.
+        # Quarkus uses this field to associate an SBOM with an erratum.
+        erratum_json = self.get(f"api/v1/erratum/{erratum_id}?format=json")
+        try:
+            # All products that use SBOMer should have notes set
+            notes = json.loads(erratum_json["content"]["content"]["how_to_test"])
+        except json.JSONDecodeError:
+            logger.warning(f"Couldn't load Notes for erratum {erratum_id}")
+            return {}
```

Can we just raise the exception here if `json.loads()` fails? We've seen issues in other tasks get hidden because we only logged a warning and skipped loading that data. That caused a lot of missing data we had to go back and reprocess after fixing the bug.